docs/topics/dokkatoo-migration.md

```diff
@@ -0,0 +1,359 @@
+[//]: # (title: Migrate to Dokka Gradle plugin v2)
+
+> The Dokka Gradle plugin v2 is an [experimental](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained) feature. 
+> It may be changed at any time. We appreciate your feedback in [GitHub](https://github.com/Kotlin/dokka/issues).
+>
+{type="warning"}
+
+The Dokka Gradle plugin (DGP) is a tool to generate comprehensive API documentation for Kotlin projects.
+
+DGP seamlessly processes both Kotlin's KDoc comments and Java's Javadoc comments to extract information and create 
+structured documentation in [HTML or Javadoc](#select-documentation-output-format) format.
+
+From Dokka 2.0.0, you can try the Dokka Gradle plugin v2, the new version of DGP.
+
+DGP v2 introduces significant improvements to DGP, aligning more closely with Gradle best practices. Key improvements are:
+
+* Adoption of Gradle types, leading to better performance.
+* Use of intuitive top-level DSL configuration instead of low-level task-based setups, simplifying the build scripts and their readability.
+* More declarative approach to documentation aggregation, making multi-project documentation easier to manage.
+* Plugin configuration is now typesafe, improving the reliability and maintainability of your build scripts.
+
+## Before you start
+
+Before diving into your project migration, complete the following preparatory steps according to your project.
```

Comment:
It's a very elaborate sentence, maybe a bit too much for docs type of content. Maybe:
> Before starting the migration, complete the following steps.